styx-common/pom.xml

```diff
@@ -137,5 +137,10 @@
       <artifactId>JUnitParams</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
```

Comment:
let's use `slf4j-simple` instead.

Reply:
Yeah, this is so that ocular inspection is possible.